sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Release History
+
+## 1.0.0-preview.1 (2020-01-09)
+
+- This release is a preview of our efforts to create a client library that is user friendly and
+  idiomatic to the JavaScript ecosystem. The reasons for most of the changes in this update can be found in the
+  [Azure SDK Design Guidelines for TypeScript](https://azuresdkspecs.z5.web.core.windows.net/TypeScriptSpec.html).
+- Differences from previous public package `@azure/cognitiveservices-textanalytics`
+  - Package name changed from `@azure/cognitiveservices-textanalytics` to `@azure/ai-text-analytics`.
```

Comment:
The package name point can be deleted as the line above implies the same
```suggestion
```

Reply:
aah, thank you so much for this change. I was working on some updates and was wondering why am I getting rush error for @azure/cognitiveservices-textanalytics when it is auto-generated, and only to realize the same package name existed in two places in the repo :(